SteamProject/MySteam/Steam.cpp

```diff
@@ -155,87 +143,133 @@ void FSteam::OpenCategoriesMenu()
 		system("cls");
 		std::cout << "-----------------Cateories Menu-----------------" << std::endl;
 		std::cout << "\t\t1 - Add category \n\t\t2 - Delete category \n\t\or Press Enter to return to the Main menu \n" << std::endl;
+
 		std::cin.clear();
 		std::cin.ignore(1000, '\n');
 		if (std::cin.peek() == '\n')
 		{
 			break;
 		}
-		CategoriesOption = ValidateInput("choose an option");
+		CategoriesOption = ValidateInput("");
```

Comment:
Continuing from the previous comment, the solution would also remove the need to pass an empty string by parameter here. This line might be confusing to read as well, as someone might interpret it's trying to validate whether "" is a proper input.